### ISSUE TITLE
Routing requests to AWS-WPS & Geoserver WPS

### ISSUE THREAD

**Issue by ccmoloney** [aodn/aws-wps#136]
Our first production version of AWS-WPS will currently only support gs:GoGoDuck aggregations - not NetcdfOutput aggregations.

It is expected that future releases will support all types of aggregations - and the aggregation codebase currently implement in Geoserver WPS will be decommissioned.

In the interim: until AWS-WPS implements all aggregation types we will need to preserve the current implementations of aggregation types not supported in AWS-WPS : NetcdfOutput (is that the only one?).

A client (the portal for instance) will need to direct gs:GoGoDuck aggregation execute requests to AWS-WPS & all other aggregation execute requests to Geoserver WPS.

Can we easily cater for this behaviour in a way that is relatively transparent to a client?  Will updating URLs in the metadata for the collections that invoke gs:GoGoDuck be sufficient to point those collections to the new AWS-WPS aggregator?

**Comment by jonescc** [comment]
The WPS service endpoints are configured in the metadata for each collection.  The portal should always direct the request to the endpoint specified in the metadata - the content of the request will not be different for the two different systems  - so hopefully we haven't broken this.   

We are meant to be making changes to the portal this iteration to support the different processing required between AWS WPS and GeoServer WPS.   For Geoserver endpoints, the portal will still need to send emails on submission and in response to callbacks from geoserver and display the status pages.  For AWS endpoints, AWS WPS will send the emails and provide status pages.

Provided we do this by detecting what type of endpoint we are talking to and support the required processing for that type of endpoint (I think this is what @pmbohm was planning) then we should be able to handle both NetcdfOutput and GoGoDuck processing.

**Comment by pmbohm** [comment]
Yes @ccmoloney is right! 
Both these protocols use an email address. My portal pull request will have to retain emailing capabilities to support `OGC:WPS--netcdf-subset-service`
```
        [ 'handler': 'GogoduckDownloadHandler',            'protocol': 'OGC:WPS--gogoduck'                   ],
        [ 'handler': 'NetcdfSubsetServiceDownloadHandler', 'protocol': 'OGC:WPS--netcdf-subset-service'      ],
```

**Comment by pmbohm** [comment]
Addressed by https://github.com/aodn/aodn-portal/pull/2571

**Comment by jonescc** [comment]
Fixed by aodn/aodn-portal#2571
